client/signup/config/flows.js

```diff
@@ -107,7 +108,10 @@ function getThankYouNoSiteDestination() {
 }
 
 function getChecklistThemeDestination( { siteSlug, themeParameter } ) {
-	if ( themeParameter === 'blank-canvas-3' ) {
+	if (
+		themeParameter === 'blank-canvas-3' &&
+		config.isEnabled( 'pattern-assembler/logged-out-showcase' )
```

Comment:
Since it affects logged-in showcase as well, maybe rename the flag to `pattern-assembler/theme-showcase` for example? 😄

Reply:
Or do we want to differentiate the behavior between logged-in and logged-out?

Reply:
For scope of this project, yes. Since this change in the signup destination.

Reply:
Yes, I think we navigate the user to PA only for the logged-out user in the current stage.